From df297cc5625e844d12e37604ed3a62afcf03b409 Mon Sep 17 00:00:00 2001
From: Howard Hsu <howard-yh.hsu@mediatek.com>
Date: Tue, 22 Apr 2025 10:48:10 +0800
Subject: [PATCH 083/119] mtk: mt76: mt7996: rework the init value for current
 tx duty cycle

The init value of current tx duty cycle is 0 because firmware will not
send the unsolicited event with current tx duty cycle every second. Add
the init value as 100 to corrent the inin tx duty cycle.

Signed-off-by: Howard Hsu <howard-yh.hsu@mediatek.com>
---
 mt7996/mcu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mt7996/mcu.c b/mt7996/mcu.c
index de621eec..fa06d2b5 100644
--- a/mt7996/mcu.c
+++ b/mt7996/mcu.c
@@ -6394,6 +6394,8 @@ int mt7996_mcu_set_thermal_throttling(struct mt7996_phy *phy, u8 state)
 			return ret;
 	}
 
+	phy->throttle_state = state;
+
 	return 0;
 }
 
-- 
2.45.2

